Formula/gofumpt.rb

```diff
@@ -0,0 +1,18 @@
+class Gofumpt < Formula
+  desc "Stricter gofmt"
+  homepage "https://github.com/mvdan/gofumpt"
+  url "https://github.com/mvdan/gofumpt/archive/v0.1.0.tar.gz"
+  sha256 "802c92d3df222c9b266d785305107c58a26ea186c4dbb5989b0db97b9bce0367"
+  license "BSD-3-Clause"
+
+  depends_on "go"
+
+  def install
+    system "go", "build", *std_go_args, "-ldflags", "-s -w -X main.version=#{version}", "-trimpath"
+  end
+
+  test do
+    output = shell_output("#{bin}/gofumpt --version")
+    assert_match "0.1.0", output
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
how can i add a test file? than i could add some better testing... since this is a linter.

Reply:
```
(testpath/"file").write <<~EOS
  file
EOS

Reply:
did that now :) thanks